interface.py

```diff
@@ -97,9 +97,14 @@ def switch_power_down(self):
 
     def confirm_selection(self):
         """
-        When indicator is False the player can choose cards and switch Power Down.
-        When is True the player ended the selection of cards.
+        When Power Down is switched on or all cards are chosen,
+        it is possible to confirm selection.
+        Additionally when Power Down is switched on,
+        the cards from hand are returned.
         """
-        if None not in self.program:
+        if None not in self.program or self.power_down:
             self.selection_confirmed = True
+            if self.power_down:
```

Comment:
přijde mi to tu nějak složité, radši bych napsala novou metodu pro power_down, nebo upravila switch power down, pokud se výběr zmáčkne, automaticky se potvrdí výběr a smažou se karty, ale poslechnu si názory jiných :) Nelíbí se mi, že si můžu mačkat power down pořád dokola než to potvrdím :-D